src/logic/exceptions/registry.ts

```diff
@@ -9,6 +9,8 @@ enum ErrorCodes {
   _100 = '100: Invalid input in the address field',
   _600 = '600: Error fetching token list',
   _601 = '601: Error fetching balances',
+  _900 = '900: Error loading Safe App',
```

Comment:
Let's not add it if it's not used yet.

Reply:
I just wanted to showcase that 9** is taken by safe apps, added the usage in the following commit